doc/DeveloperGuide.md

```diff
@@ -88,6 +88,29 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Rename person
+
+**MSS**
+1. User requests to rename person
+2. User enters name of person to be renamed and new name of the person to be renamed.
```

Comment:
Use case sequence alternates between the actor and the system.
Your current case is actor -> actor -> system. 

A better use case would be:
`1. User requests to rename person`
**`2. Addressbook requests for name of person.`**
`3. User enters name of person to be renamed and new name of the person to be renamed.`
`...`